dotnetv3/S3/versioned-examples/DeleteMultipleObjectsExample/DeleteMultipleObjectsExample/DeleteMultipleObjects.cs

```diff
@@ -0,0 +1,253 @@
+﻿namespace DeleteMultipleObjectsExample
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
+    /// <summary>
+    /// This example shows how to delete objects in a version-enabled Amazon
+    /// Simple StorageService (Amazon S3) bucket. It was created using AWS
+    /// SDK for .NET version 3.7 and .NET Core 5.0.
+    /// </summary>
+    public class DeleteMultipleObjects
+    {
+        public static async Task Main()
+        {
+            string bucketName = "doc-example-bucket";
+
+            // If the AWS region for your Amazon S3 bucket is different from
+            // the AWS region of the default user, define the AWS Region for
```

Comment:
AWS region -> AWS Region (for consistancy)

Reply:
fixed